jongseong/section03/api/main.py

```
@@ -0,0 +1,26 @@
+from fastapi import FastAPI, UploadFile, File
+
+app = FastAPI()
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+# 임시 파일 저장 경로 설정
+FILE_DIR = Path(BASE_DIR / "files")
+FILE_DIR.mkdir(exist_ok=True)
+
+
+@app.post("/upload/", description="파일 업로드")
+async def upload_file(file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)